config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  # resources :apartments
+  resources :apartments
+  resources :apartments
```

Comment:
You have resources for apartments twice.  You should remove one and double check that the routes work properly for apartments (rails routes should tell you what you need to know)